tests/test_plugins.py

```diff
@@ -49,7 +53,7 @@ def test_signal_connection(qapp, qtbot):
     # Try changing types
     qapp.processEvents()
     qapp.processEvents()  # Must be called twice. Multiple rounds of signals
-    sig_conn.remove_listener(listener)
+    qapp.close_widget_connections(widget)
```

Comment:
Maybe it would be wise to also wrap this with the check for the `hasattr`?
We will deprecate the `close_widget_connections` at the qapp in a couple of releases.
For now it will work but it can start failing later on.

Reply:
👍